aapcs64/aapcs64.rst

```diff
@@ -892,6 +892,8 @@ thread-local storage on platforms where multi-threaded code is
 supported.  The exact location of such information is platform
 specific.
 
+The FPMR is system register that controls behaviors of the FP8 instructions. It is a temporary register. The state of the register must be held in thread-local storage on platforms where multi-threaded code is supported.
```

Comment:
I believe you need to manually cut the line for 80 characters.

Reply:
I would very much prefer that was a  part of the "coding standard" for ABI.